modules/manage/partials/monitor-health.adoc

```diff
@@ -4,6 +4,16 @@ This section provides guidelines and example queries using Redpanda's public met
 
 TIP: To help detect and mitigate anomalous system behaviors, capture baseline metrics of your healthy system at different stages (at start-up, under high load, in steady state) so you can set thresholds and alerts according to those baselines.
 
+[TIP]
```

Comment:
@wzzzrd86 I'd like to add the new admonition around here, except there is the existing tip ^ about capturing baseline metrics. This is the current preview if we put the two admonitions right next to each other: https://deploy-preview-913--redpanda-docs-preview.netlify.app/current/manage/monitoring/#monitor-for-performance-and-health can we change or downgrade one of them to just be in regular paragraph formatting, or do they both have to be called out?

Reply:
@kbatuigas would it be in the same area, just not marked as a Tip?  If so I think that would be fine.

Reply:
Removed the admonition for the existing tip about baseline metrics, it flows better with the first paragraph whereas the info about counter metrics resetting seems a bit more extraneous